### ISSUE TITLE
False positive on `X if cond else Y`

### ISSUE THREAD

**Issue by nyuszika7h** [jsfehler/flake8-multiline-containers#114]
The following code falsely triggers a JS101 error:

```py
x = {} if y == 1 else { 
    "foo": 42,          
}                       
```

The first container here isn't multiline, and the second is broken properly.

**Comment by jsfehler** [comment]
Fixed by #122
